docs/pages/authentication/overview.mdx

```diff
@@ -0,0 +1,136 @@
+
+## Why Authentication Matters in Web3
+
+Traditional blockchain authentication creates significant barriers to user adoption:
+
+- **Complex private key management** - Users must securely store and manage cryptographic keys
+- **Wallet installation requirements** - Additional software downloads and setup
+- **Technical knowledge barriers** - Understanding of blockchain concepts and wallet security
+- **Poor user experience** - Lengthy onboarding flows and confusing interfaces
+
+Alchemy Smart Wallets solves these challenges by providing **web2-familiar authentication** that creates **web3-native smart accounts** behind the scenes.
+
+## Core Authentication Features
+
+### Multiple Authentication Methods
+
+Choose from a variety of authentication options to match your users' preferences:
+
+- **[Email OTP](/wallets/authentication/login-methods/email-otp)** - Passwordless login with 6-digit verification codes
+- **[Email Magic Links](/wallets/authentication/login-methods/email-magic-link)** - One-click authentication via secure email links
+- **[Social Login](/wallets/authentication/login-methods/social-login)** - OAuth with Google, Apple, Discord, Twitter, and more
+- **[Passkey Authentication](/wallets/authentication/login-methods/passkey-signup)** - Biometric and hardware-based passwordless login
+- **[Bring Your Own Auth](/wallets/authentication/login-methods/bring-your-own-auth)** - Integrate existing JWT/OIDC authentication systems
+
+### Flexible Implementation Options
+
+Implement authentication in the way that best fits your application:
+
+- **Pre-built UI Components** - Deploy quickly with minimal code using `<AuthCard />` or `<AuthModal />`
+- **Custom UI Integration** - Build completely custom experiences with headless React hooks
+- **Modal or Embedded Flows** - Choose between popup modals or embedded authentication forms
+- **Whitelabel Solutions** - Customize theming and branding to match your application
+
+### Enterprise-Grade Security
+
+- **Non-custodial Architecture** - Users maintain full control over their private keys
+- **Secure Enclave Storage** - Keys generated and stored in hardware security modules
+- **Multi-Factor Authentication (MFA)** - Additional security layers for high-value operations
+- **Account Recovery** - Flexible recovery options through trusted contacts or backup methods
+
+## Understanding Signers
+
+A **Signer** is the service or application that manages private keys and signs operations for your users. With Alchemy Smart Wallets:
+
+- **Alchemy Signer** - Our secure, non-custodial signer service powered by Turnkey's secure enclaves
+- **Third-Party Signers** - Integration with existing wallet providers like MetaMask
+- **User Control** - Only end users can access their keys, not Alchemy or your application
+
+Learn more: **[What is a Signer?](/wallets/authentication/what-is-a-signer)**
```

Comment:
Does this page exist (https://www.alchemy.com/docs/wallets/authentication/what-is-a-signer)? Just looking to see where we send folks, not sure if its coming in another PR